uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Import/Parser/Base.php

```diff
@@ -94,6 +100,8 @@ public function init() {
     $this->setActiveFields($this->_mapperKeys);
     $this->absenceTypes = $this->getAbsenceTypes();
     $this->absenceStatuses = $this->getAbsenceStatuses();
+    $session = CRM_Core_Session::singleton();
+    $this->dateFormatType = $session->get('dateTypes');
```

Comment:
Just to confirm, will `$session->get('dateTypes')` return a single value, despite the fact that it is using the plural form?